Update docker-images to version 79

ebyhr opened the pull request:
## Description

Update docker-images to version 79

## Release notes

(x) This is not user-visible or docs only and no release notes are required.

findinpath commented:
Tests are red 
```
2023-04-07 11:32:20 INFO: FAILURE     /    io.trino.tests.product.iceberg.TestIcebergPartitionEvolution.testDroppedPartitionField [true] (Groups: profile_specific_tests, iceberg) took 3.0 seconds
tests               | 2023-04-07 11:32:20 SEVERE: Failure cause:
tests               | java.lang.AssertionError: Could not find rows:
tests               | [a, null, null, 0.16666666667, null, null, null]
tests               | [b, null, null, 0.16666666667, null, null, null]
tests               | [c, null, null, 0.00000000000, null, null, null]
tests               | 
tests               | actual rows:
tests               | [a, null, 3.00000000000, 0.16666666667, null, null, null]
tests               | [b, null, 3.00000000000, 0.16666666667, null, null, null]
tests               | [c, null, 4.00000000000, 0.00000000000, null, null, null]
tests               | [null, null, null, null, 6.00000000000, null, null]
```

ebyhr commented:
The change is different from https://github.com/trinodb/trino/pull/16557 and the new row is the expected result in my understanding.

findinpath commented on testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksChangeDataFeedCompatibility.java:
```diff
@@ -52,7 +53,7 @@ public void setup()
     }
 
-    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_EXCLUDE_73, PROFILE_SPECIFIC_TESTS})
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, DELTA_LAKE_EXCLUDE_73, PROFILE_SPECIFIC_TESTS})
```
If we add `DELTA_LAKE_OSS` on all the methods, we may as well change the class name to `TestDeltaLakeChangeDataFeedCompatibility`.

findinpath commented:
Can it be that what we've found in https://github.com/trinodb/trino/pull/16921#issuecomment-1500139434 is a side-effect of bf04a728 ?

I wonder why it didn't pop up before ... 

cc @findepi

ebyhr commented:
> Can it be that what we've found in https://github.com/trinodb/trino/pull/16921#issuecomment-1500139434 is a side-effect of https://github.com/trinodb/trino/commit/bf04a72875f27d43301d7896b6e04f0bc1199a6d ?

I don't think so. I suppose `ALTER TABLE ... DROP PARTITION FIELD` on old Iceberg Spark didn't handle puffin files correctly (or did something wrong) and then `SHOW STATS` returned incorrect results.

ebyhr commented:
The following failure looks an actual issue. 
```
2023-04-07T14:55:57.2405351Z tests               | 2023-04-07 20:40:57 INFO: FAILURE     /    io.trino.tests.product.iceberg.TestIcebergSparkCompatibility.testPartialStats (Groups: iceberg_jdbc, profile_specific_tests, iceberg, iceberg_rest) took 1.3 seconds
2023-04-07T14:55:57.2406262Z tests               | 2023-04-07 20:40:57 SEVERE: Failure cause:
2023-04-07T14:55:57.2407066Z tests               | io.trino.tempto.query.QueryExecutionException: java.sql.SQLException: org.apache.hive.service.cli.HiveSQLException: Error running query: org.apache.iceberg.exceptions.CommitStateUnknownException: Service failed: 500: {
2023-04-07T14:55:57.2407910Z tests               | "cause1":"java.lang.IllegalArgumentException: Cannot parse missing map: updated",
2023-04-07T14:55:57.2408961Z tests               | "servlet":"org.apache.iceberg.rest.RESTCatalogServlet-6385cb26",
2023-04-07T14:55:57.2410434Z tests               | "cause0":"com.fasterxml.jackson.databind.JsonMappingException: Cannot parse missing map: updated (through reference chain: org.apache.iceberg.rest.requests.UpdateTableRequest[&quot;updates&quot;]-&gt;java.util.ArrayList[0])",
2023-04-07T14:55:57.2411843Z tests               | "message":"com.fasterxml.jackson.databind.JsonMappingException: Cannot parse missing map: updated (through reference chain: org.apache.iceberg.rest.requests.UpdateTableRequest[&quot;updates&quot;]-&gt;java.util.ArrayList[0])",
2023-04-07T14:55:57.2412603Z tests               | "url":"/v1/namespaces/default/tables/test_partial_stats_sznstttwsl",
2023-04-07T14:55:57.2412942Z tests               | "status":"500"
```

findinpath commented:
@ebyhr  pls update 

https://github.com/trinodb/trino/blob/20841a66fd7b3c11dbaa2754ac1c0da31322dcd4/testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvSinglenodeSparkIcebergRest.java#L46

to 
```
    private static final String REST_SERVER_IMAGE = "tabulario/iceberg-rest:0.3.0";
```

which will solve the failing test `TestIcebergSparkCompatibility.testPartialStats`

ebyhr commented:
@findinpath Thanks!

Let me continue in #16957 to run with secrets.
